app/src/main/java/com/hedvig/app/util/ToolbarHeight.kt

```diff
@@ -0,0 +1,14 @@
+package com.hedvig.app.util
+
+import android.util.TypedValue
+import androidx.fragment.app.Fragment
+
+
+fun getToolbarBarHeight(context: Fragment): Int {
```

Comment:
Won't the size of the toolbar be dependent on its interactions with the system window insets?

I'm afraid that this code might be a bit brittle.